docs/source/howto/level1-csd.rst

```diff
@@ -0,0 +1,312 @@
+
+    force-level1-csd meta gcs_berlin/level1 gcs_berlin/level1/l1_pool.txt T33UUU,T33UVU -n -c 0,70 -d 20190101,20191231 -s S2B
+
+    Querying the metadata catalogue for Sentinel2 data
+    Sensor(s): S2B
+    Tile(s): T33UUU,T33UVU
+    Daterange: 20190101 to 20190430
+    Cloud cover minimum: 0%, maximum: 70%
+
+    44 Sentinel2 Level 1 scenes matching criteria found
+    21.06GB data volume found.
+
+    Done.
+
+
+When we're happy with the results, we can remove the ``-n | --no-act`` option to start downloading
+
+.. code-block:: none
+
+    force-level1-csd meta gcs_berlin/level1 gcs_berlin/level1/l1_pool.txt T33UUU,T33UVU -c 0,70 -d 20190101,20191231 -s S2B
+
+    Querying the metadata catalogue for Sentinel2 data
+    Sensor(s): S2B
+    Tile(s): T33UUU,T33UVU
+    Daterange: 20190101 to 20190430
+    Cloud cover minimum: 0%, maximum: 70%
+
+    44 Sentinel2 Level 1 scenes matching criteria found
+    21.06GB data volume found.
+
+    Starting to download 44 Sentinel2 Level 1 scenes
+
+
+    Downloading S2B_MSIL1C_20190119T101339_N0207_R022_T33UVU_20190119T121912(5 of 44)...
+
+    |===========                                                                            |  13 %
+
+
+The status bar will update after every scene that has successfully been downloaded. Any scenes that have already been downloaded will be skipped and a notification will be printed. If the download was canceled at some point and a scene wasn't fully downloaded, FORCE Level 1 CSD will delete that scene and redownload it.
+
+
+A note on Sentinel-2 scenes
+"""""""""""""""""""""""""""
+
+The processing baseline for Landsat data is standardized by the Collection format. For Sentinel-2 we found that there were duplicate scenes with different processing baselines or processing dates. FORCE Level 1 CSD will check for duplicates and only download the version with the highest processing baseline or latest processing date if the processing baseline is the same.
+
+
+Saving metadata
+---------------
+
+FORCE Level 1 CSD offers the possibility to save the metadata of the scenes that matched the specified requirements. Having easy access to this kind of information can be very helpful when creating statistics and visualizations about data availability over time, cloud cover distribution over time, data volume (e.g., per sensor or year), etc. Using the ``-k | --keep-meta`` option will save the results of the current query to the Level 1 datapool folder under the file name
+
+.. code-block:: none
+
+    csd_metadata_YYYY-MM-DDTHH-MM-SS.txt
+
+
+The ``-k | --keep-meta`` option can be combined with the ``-n | --no-act`` option to acquire the metadata without downloading any data.
+If Landsat and Sentinel-2 data is queried at the same time, two separate files will be created. Please note that Google's metadata catalogues use different column names for Landsat and Sentinel-2.
+
+
+Scenes from the same path/row (Landsat) or MGRS tile (Sentinel-2) are stored in folders
```

Comment:
Add a bash code snippet here à la

ls -l meta

Would probably even fit better into the "-u" block at the beginning of this tutorial